test/server/server_fuzz_test.cc

```diff
@@ -62,6 +62,10 @@ makeHermeticPathsAndPorts(Fuzz::PerTestEnvironment& test_env,
     for (auto& host : *cluster.mutable_hidden_envoy_deprecated_hosts()) {
       makePortHermetic(test_env, host);
     }
+    // Not implemented
+    if (cluster.lb_policy() == envoy::config::cluster::v3::Cluster::LOAD_BALANCING_POLICY_CONFIG) {
+      cluster.set_lb_policy(envoy::config::cluster::v3::Cluster::ROUND_ROBIN);
```

Comment:
Did you consider PGV annotations? The advantage there is we would also block these configs in the production Envoy server.

Reply:
Sure -- I could potentially add in a constraint for the enum to be one of the defined values, but that would make the API more restrictive 
https://github.com/envoyproxy/envoy/blob/b9aff554ca3a574b9eb5df96cd7a7a325fa4d0f5/api/envoy/config/cluster/v3/cluster.proto#L591

But given that it's enforced in code I'm happy to use this pattern! There are some other examples that are WIP that I'll change to this pattern as well.